core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -544,8 +544,6 @@
       "Properties in this category affect the behavior of the monitor web server."),
   MONITOR_PORT("monitor.port.client", "9995", PropertyType.PORT,
       "The listening port for the monitor's http service"),
-  MONITOR_LOG4J_PORT("monitor.port.log4j", "4560", PropertyType.PORT,
-      "The listening port for the monitor's log4j logging collection."),
```

Comment:
It looks like this property was only used in MiniAccumulo?

Reply:
Users could have used it, but it's a NOOP now.